app/views/merchant_items/index.html.erb

```diff
@@ -1,10 +1,15 @@
-<% @merchant.items.each do |item| %>
-  <div id="item-<%= item.id %>">  
-    <%= link_to "#{item.name}", "/merchants/#{@merchant.id}/items/#{item.id}" %>
-    <% if item.status == "disabled" %>
-      <%= button_to "Enable" ,"/merchants/#{@merchant.id}/items/#{item.id}", method: :patch, local: true, params: {status: "enabled"} %>
-    <% elsif item.status == "enabled" %>
-      <%= button_to "Disable" ,"/merchants/#{@merchant.id}/items/#{item.id}", method: :patch, local: true, params: {status: "disabled"} %>
-    <% end  %>
-  </div>
-<% end %>
+<h1>Items for Merchant: <%= @merchant.name %></h1>
+
+<div id="Enabled-items">
+  <h3>Enabled Items</h3>
+  <% @merchant.filter_item_status(1).each do |item| %>
+    <%= link_to "#{item.name}", "/merchants/#{@merchant.id}/items/#{item.id}" %><%= button_to "Disable" ,"/merchants/#{@merchant.id}/items/#{item.id}", method: :patch, local: true, params: {status: "disabled"} %>
```

Comment:
I like your use of the extra params in your button.  I thought this section in general was nicely done.  All the work you did looks appropriate for a View and is easy to understand what you want from it. I also liked that you added <div> to take advantage of within blocks for your tests